ts/components/messages/MessageDetailCTABar.tsx

```diff
@@ -48,15 +56,20 @@ class MessageDetailCTABar extends React.PureComponent<Props> {
     return fromNullable(this.props.message.content.due_date);
   }
 
+  public componentDidMount() {
+    if (!this.props.serviceMetadata && this.props.service) {
+      this.props.loadService(this.props.service);
```

Comment:
```suggestion
      this.props.loadServiceMetadata(this.props.service);
```
Could it be a better name?